src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -338,6 +339,53 @@ class RequiredMandatoryAttribute:
                         success = False
                         continue
 
+        # Validate there are attributes in the global range that are not in the required list
+        allowed_globals = [a.id for a in ATTRIBUTES_TO_CHECK]
+        # also allow event list because it's not disallowed
+        event_list_id = 0xFFFA
+        allowed_globals.append(event_list_id)
+        global_range_min = 0x0000_F000
+        standard_range_max = 0x000_4FFF
+        mei_range_min = 0x0001_0000
+        attribute_list_id = 0xFFFB
```

Comment:
I assume we could just use ATTRIBUTE_LIST_ID here?

Reply:
thanks, yeah. Good call - changed.